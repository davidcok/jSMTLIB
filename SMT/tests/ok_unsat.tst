(set-logic QF_UF)
(declare-fun p () Bool)
(declare-fun q () Bool)
(assert (or p q))
(assert (not p))
(assert (not q))
(check-sat)
