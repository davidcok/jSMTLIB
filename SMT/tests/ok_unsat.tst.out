success
success
success
success
success
success
unsat
